src/server/JasmineGraphServer.cpp

```diff
@@ -3223,18 +3233,9 @@ void JasmineGraphServer::initiateMerge(std::string graphID, std::string training
     map<string, map<int, int>> scheduleForAllHosts = schedular->schedulePartitionTraining(graphID);
     std::map<std::string, JasmineGraphServer::workerPartition> graphPartitionedHosts = this->getWorkerPartitions(
             graphID);
-    int partition_count = 0;
-    std::map<std::string, JasmineGraphServer::workerPartition>::iterator mapIterator;
-    for (mapIterator = graphPartitionedHosts.begin(); mapIterator != graphPartitionedHosts.end(); mapIterator++) {
-        JasmineGraphServer::workerPartition workerPartition = mapIterator->second;
-        std::vector<std::string> partitions;
-        partitions.push_back(workerPartition.partitionID);
-        std::vector<std::string>::iterator it;
-        for (it = partitions.begin(); it < partitions.end(); it++) {
-            partition_count++;
-        }
-    }
+    int partition_count = graphPartitionedHosts.size();
 
+    std::cout << partition_count << std::endl;
```

Comment:
Lets remove these `std::cout` statements. If those are really required lets convert those to log statements.

Reply:
fixed

Reply:
Ok

Reply:
Ok